crypto/evp_extra/p_pqdsa_asn1.c

```diff
@@ -18,6 +18,40 @@ static void pqdsa_free(EVP_PKEY *pkey) {
   pkey->pkey.pqdsa_key = NULL;
 }
 
+static int pqdsa_get_priv_raw_seed(PQDSA_KEY *key, const PQDSA *pqdsa,
+                                  uint8_t *out,size_t *out_len) {
+  if (key->seed == NULL) {
+    OPENSSL_PUT_ERROR(EVP, EVP_R_NO_KEY_SET);
+    return 0;
+  }
+
+  if (*out_len != key->pqdsa->keygen_seed_len) {
```

Comment:
```suggestion
  if (*out_len < key->pqdsa->keygen_seed_len) {
```
A buffer should be allowed to be bigger I'd imagine? The point is that we set `*out_len` the actual value of what we wrote. Though I do realize this i gated later, I don't know if it needs to be gated here per-se.

Reply:
Thank you, added in [c307952](https://github.com/aws/aws-lc/pull/2194/commits/c30795274cf00bed09803b8f5fe6114112fb1422)